### ISSUE TITLE
Quote rendering

### ISSUE THREAD

**Issue by mrzv** [charmbracelet/glamour#172]

**Comment by NReilingh** [comment]
+1 here as well. Note also that line breaks inside of a blockquote are affected by #84.

[This CommonMark dingus][1] illustrates the correct rendering of markdown blockquotes with regard to line breaks.

[1]: https://spec.commonmark.org/dingus/?text=%3E%20This%20is%20a%20blockquote%0A%3E%20that%20uses%20semantic%20linefeed.%0A%3E%20All%20of%20the%20lines%20are%20flowed%20to%20the%20width%20of%20the%20viewport.%0A%0A%3E%20This%20is%20also%20a%20blockquote%0Awhich%20is%20rendered%20the%20same%20way.%0ANote%20that%20the%20%60%3E%60%20prefix%20is%20optional%20on%20each%20line.%0A%0A%3E%20However%2C%20recognize%20that%20if%20a%20%20%0A%3E%20hard%20break%20is%20used%20(the%20above%20line%20ends%20with%20two%20spaces)%2C%0A%3E%20the%20blockquote%20should%20have%20a%20linebreak%20rendered.%0A%0A%0A

**Comment by pjeby** [comment]
I don't think this is limited to block quotes - I'm seeing this orphan word issue with words that look like they would've just exactly fit on the previous line, in normal text.

Also, if you tweak the value of `-w` with the blockquote, you can see it's a line-length issue: a -w of 1 greater or lesser than the width where it breaks will work.  So if `-w80` breaks at some spot, `-w79` or `-w81` will be fine at that spot for the same text.  So it  appears to be a fencepost error in the wrapping logic, where it calculates one place it'll fit on the same line and then in another place says, nuh uh and wraps it.

**Comment by brontosaurusrex** [comment]
Similar on pi5/arm
(glow version 2.1.2 (f570874))

```
glow -w 36

  https://en.wikipedia.
  org/wiki/Zettelkasten

  │ A Zettelkasten (German:
  │ 'slipbox', plural
  Zettelkästen)
  │ or card file consists of small
  │ items of information stored on
  │ Zetteln (German: 'slips';
  │ singular: Zettel), paper slips
  │ or cards, that may be linked
  to
  │ each other through subject
  │ headings or other metadata
  such
  │ as numbers and tags. It has
  │ often been used as a system of
  │ note-taking and personal
  │ knowledge management for
  │ research, study, and writing.
```
